endpoints/openrtb2/auction_test.go

```diff
@@ -197,15 +274,36 @@ func readFile(t *testing.T, filename string) []byte {
 }
 
 // doRequest populates the app with mock dependencies and sends requestData to the /openrtb2/auction endpoint.
-func doRequest(t *testing.T, requestData []byte, aliased bool) (int, string) {
+func (gr *getResponseFromDirectory) doRequest(t *testing.T, requestData []byte) (int, string) {
 	aliasJSON := []byte{}
-	if aliased {
+	if gr.aliased {
 		aliasJSON = []byte(`{"ext":{"prebid":{"aliases": {"test1": "appnexus", "test2": "rubicon", "test3": "openx"}}}}`)
 	}
+	disabledBidders := map[string]string{
```

Comment:
This is a lot of copy/paste code from the `router`... should try to find some way to share it, if it's possible.

I won't have time to think much about it before vacation... but this is usually a sign that some code lives in a module where it doesn't belong, or is private and ought to be public so other modules can use it.